tasties/urls.py

```diff
@@ -25,5 +25,5 @@
     path('login/', views.login_user, name="login"),
     path('register/', views.register, name="register"),
     path('logout/', views.logout_user, name="logout"),
-    path('view_recipe/', views.view_recipe, name="view_recipe")
+    path('view_recipe/<str:recipe_title>/', views.view_recipe, name="view_recipe")
```

Comment:
Why not use the recipe id here instead?

Reply:
+1

Reply:
Changes the URL to use the id instead of the title.
Added some more tests.

Reply:
Great! think that you can't rely upon your titles will be always in English so using IDs in the URLs is always preferred